apps/src/amazonFutureEngineerEligibility/amazonFutureEngineerEligibility.js

```diff
@@ -10,12 +10,25 @@ function showAmazonFutureEngineerEligibility() {
     '.amazon-future-engineer-eligibility-container'
   );
 
-  ReactDOM.render(
-    // TO DO: confirm that there's no case where this might be used
-    // where amazonFutureEngineerEligibilityElement could be null or not an array.
-    <AmazonFutureEngineerEligibility />,
-    amazonFutureEngineerEligibilityElement[0]
-  );
+  let signedIn = false;
+  $.ajax({
+    type: 'GET',
+    url: '/dashboardapi/v1/users/me/contact_details'
```

Comment:
This is a somewhat arbitrary API call. Could create a new one that explicitly returns a true/false for whether a user is signed in, but I thought we might want to do something like this to get a user's email/school and pre-populate the form anyway.